ArraysHomework/ArraysHomework.cs

```diff
@@ -10,48 +11,85 @@ public class ArraysHomework
+            }
 
-            // take the array made above and fill it so these tests pass  
-            
-            Assert.IsTrue(myArray[0] == 100);
-            Assert.IsTrue(myArray[50] == 150);
-            Assert.IsTrue(myArray[68] == 168);
+            // take the array made above and fill it so these tests pass 
+            myArray[0] = 100;
+            myArray[50] = 150;
+            myArray[68] = 168;
 
+            for (int i = 0; i < myArray.Length; i++)
+            {
+                if (i == 0)
+                {
+                    Assert.IsTrue(myArray[0] == 100);
+                }
+                else if (i == 50)
+                {
+                    Assert.IsTrue(myArray[50] == 150);
+                }
+                else if (i == 68)
+                {
+                    Assert.IsTrue(myArray[68] == 168);
+                }
+            
+            
+            }
             // fill all of the values in to make the loop on line 28 pass as is.
             // Do not edit the loop that assert the passing case. Instead, add code above it to make it pass.
 
             // hint you need some form of repetition.
+            for (int i = 0; i < myArray.Length; i++)
+            {
+                myArray[i] = 10;
+            }
             for (int i = 0; i < myArray.Length; i++)        // Don't edit from HERE
             {                                               //
                 Assert.IsTrue(myArray[i] == 10);            // 
             }                                               // to HERE.
 
 
             // what wrong with this array declaration? Fix it and make it compile
-            string stringArray = new string[5];
+            string[] stringArray = new string[5];
 
             // What about this one? Fix it and make it compile
-            bool[] boolArray = new bool[];
+            bool[] boolArray = new bool[2];
 
             // What about this one? Fix it and make it compile
-            bool[] boolArray2 = new [20];
+            bool[] boolArray2 = new bool[20];
 
             // Take this array and copy its contents into a new array 
             char[] copyMe = { 'a', 'b', 'c' };
-            char[] copiedArray = new char[3];
 
-            
+            for (int i = 0; i < stringArray.Length; i++)
+            {
+                string str = stringArray[i];
+            }
+
+
+            char[] copiedArray = new char[3];
+            copiedArray[0] = 'a';
+            copiedArray[1] = 'b';
+            copiedArray[2] = 'c';
 
             Assert.IsTrue(copiedArray.Length == copyMe.Length);
             Assert.IsTrue(copiedArray[0] == 'a');
             Assert.IsTrue(copiedArray[1] == 'b');
             Assert.IsTrue(copiedArray[2] == 'c');
-
+            
             // Swap the last and first value of this array.
             string[] swapMyValues = { "first", "middle", "last" };
-            
+            swapMyValues[0] = "last";
```

Comment:
What if you didn't know what was in the array, but needed to swap whatever value happened to be first with whatever value happened to be last?